CRM/Core/PseudoConstant.php

```diff
@@ -1499,7 +1499,8 @@ public static function renderOptionsFromTablePseudoconstant($pseudoconstant, &$p
     $select = 'SELECT %1 AS id, %2 AS label';
     $from = 'FROM %3';
     $wheres = [];
-    $order = 'ORDER BY %2';
+    $order = 'ORDER BY %2, id';
+    // Example: 'ORDER BY abbreviation, id' because abbreviations are not unique.
```

Comment:
Maybe check that the id is in the fields? Import tables do not have an `_id` field -

It looks like you can use some variant of

```
   if (in_array('id', $availableFields), TRUE) {
     ...
    }
```

Reply:
@eileenmcnaughton Good idea. Updated.

Also, rebased on top of current master. (Since the related PR was merged, the diff should now look smaller.)